test/util/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from "./keyring";
 export * from "./polkadot-api";
+export * from "./constants";
```

Comment:
I see this file has no content... 🤔

Reply:
That's correct, i had hard coded some values there but i no longer needed it. We'll need constants though saved somewhere so keeping it here as a reminder.